src/api/app/models/bs_request_action.rb

```diff
@@ -663,9 +663,34 @@ def check_action_permission!(skip_source = nil)
     check_permissions!
   end
 
-  def expand_targets(ignore_build_state)
+  def expand_targets(ignore_build_state, ignore_delegate)
     # expand target_package
 
+    # target delegation handling
+    if action_type == :submit
```

Comment:
we check two times if the action type is submit. Look line 694:

```if action_type.in?([:submit, :maintenance_incident])```

Reply:
Yeah but this is just for submit requests, not for maintenance incident requests.